src/google/protobuf/compiler/objectivec/objectivec_generator.cc

```diff
@@ -190,10 +190,20 @@ bool ObjectiveCGenerator::GenerateAll(
       // header search path since the generate #import will be more complete.
       generation_options.runtime_import_prefix =
           StripSuffixString(options[i].second, "/");
+    } else if (options[i].first == "default_objc_class_prefix") {
+      // The default objc class prefix to use if specified by the command line 
+      // invocation. The file option is always honored first if one is present.
+      std::string value = options[i].second;
+      if (value.empty()) {
+        *error = "error: default_objc_class_prefix cannot be empty.";
+        return false;
+      }
+      SetDefaultObjcClassPrefix(options[i].second);
```

Comment:
s/options[i].second/value/

since you already fetched it.

Reply:
argh fixing.

Reply:
fixed